src/components/Activities/Activity3/Game/index.tsx

```diff
@@ -2,13 +2,16 @@ import React, { useContext, useEffect, useState } from 'react';
 import { CarouselContext } from '../../../shared/Carousel';
 import { useStateCallback } from '../../../shared/hooks';
 import DemoNextButton from './DemoNextButton';
+import FeaturesSlidebar from './FeaturesSlidebar';
 import { A3_GAME_STATE, NEXT_STATE_MAP, ONE_TIME_STATES, SESSION_CURRENT_STATE, SESSION_SKIP_STATES, SESSION_VARIABLES, VARIABLES } from './GameConstants';
 import PriorityChoices from './PriorityChoices';
 
 interface IGameContext {
   setState: (state: A3_GAME_STATE) => void,
   goNextState: () => void,
   variableSelection: VARIABLES[],
+  // featureWeights: number[],
+  // setFeatureWeights: (newWeights: number[]) => void;
```

Comment:
we want other parts of game to have access to `featureWeights`, but this is the only place where we can `setFeatureWeights`
```suggestion
  featureWeights: number[],
```